dev-requirements.txt

```
@@ -2,3 +2,4 @@ flake8
 pytest
 pcdsdaq
 pcdsdevices
+matplotlib
```

I feel so petty suggesting this, but the file was originally in alphabetical order
```suggestion
matplotlib
pytest
pcdsdaq
pcdsdevices
```

---

Of all the things flake8 doesn't clean up for me 😢